src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -25,60 +25,41 @@
 //
 //------------------------------------------------------------------------------
 
+using Microsoft.IdentityModel.Logging;
 using System.Collections.Generic;
-using System.Diagnostics.Tracing;
-using System.Globalization;
+using System.ComponentModel;
 using System.Security.Claims;
 using System.Security.Cryptography.X509Certificates;
 using System.Text;
 using System.Text.RegularExpressions;
-using System.Xml;
-using Microsoft.IdentityModel.Logging;
 using System.Threading;
+using System.Xml;
 
 namespace System.IdentityModel.Tokens.Jwt
 {
     /// <summary>
-    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Json Web Tokens. See http://tools.ietf.org/html/draft-ietf-oauth-json-web-token-07.
+    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Json Web Tokens. See: http://tools.ietf.org/html/rfc7519 and http://www.rfc-editor.org/info/rfc7515
     /// </summary>
     public class JwtSecurityTokenHandler : SecurityTokenHandler, ISecurityTokenValidator
     {
         private delegate bool CertMatcher(X509Certificate2 cert);
 
-        // the Sts pipeline expects the first identifier to be a string that 
-        // Uri.TryCreate( tokenIdentifiers[0], UriKind.Absolute, out result ) will be true.
-        // if that is not true, sts's using the .Net sts class will start failing.
-
-        private static IDictionary<string, string> outboundAlgorithmMap = new Dictionary<string, string>() 
```

Comment:
@PinpointTownes they were removed when we extended AsymmetricSignatureProvider: GetHashAlgorithmName, IsSupportedAlgorithm to recognize the long and short names. We are trying to reduce the number of 'mappings'. As we felt they were yet another layer that introduces dependencies with little ROI.

Can you let us know how we break you?  Sorry for the inconvenience :-)